gallery/gallery/lib/studies/fortnightly/app.dart

```diff
@@ -0,0 +1,163 @@
+// Copyright 2019 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:gallery/data/gallery_options.dart';
+import 'package:gallery/layout/adaptive.dart';
+import 'package:gallery/layout/text_scale.dart';
+import 'package:gallery/l10n/gallery_localizations.dart';
+import 'package:gallery/studies/fortnightly/shared.dart';
+
+const fortnightlyTitle = 'Fortnightly';
+
+class FortnightlyApp extends StatelessWidget {
+  const FortnightlyApp({Key key, this.navigatorKey}) : super(key: key);
+
+  final GlobalKey<NavigatorState> navigatorKey;
+
+  @override
+  Widget build(BuildContext context) {
+    final home = isDisplayDesktop(context)
+        ? _FortnightlyHomeDesktop()
+        : _FortnightlyHomeMobile();
+    return MaterialApp(
+      navigatorKey: navigatorKey,
+      debugShowCheckedModeBanner: false,
+      theme: buildTheme(context).copyWith(
+        platform: GalleryOptions.of(context).platform,
+      ),
+      home: ApplyTextOptions(child: home),
+      // L10n settings.
+      localizationsDelegates: GalleryLocalizations.localizationsDelegates,
+      supportedLocales: GalleryLocalizations.supportedLocales,
+      locale: GalleryOptions.of(context).locale,
+    );
+  }
+}
+
+class _FortnightlyHomeMobile extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      drawer: Drawer(
+        child: SafeArea(
+          child: NavigationMenu(isCloseable: true),
+        ),
+      ),
+      appBar: AppBar(
+        title: Semantics(
+          label: fortnightlyTitle,
+          child: Image.asset(
+            'assets/fortnightly/fortnightly_title.png',
+            excludeFromSemantics: true,
+          ),
+        ),
+        actions: [
+          IconButton(
+            icon: const Icon(Icons.search),
+            tooltip: GalleryLocalizations.of(context).shrineTooltipSearch,
```

Comment:
maybe rename this to tooltipSearch since its not used by only shrine

Reply:
The tool for importing/exporting translations gets confused if these are renamed, so I'm going to keep this as-is for now.